### PR TITLE
Fix a bunch more tests to run in STRICT mode

### PR DESCRIPTION
**Pull request by sbc100**
Tested with `./tests/runner.py strict`.  There are still a few
that need more work but this covers the low hanging fruit.

### CODE REVIEW

File: src/support.js [hunk 1]
```diff
@@ -8,28 +8,6 @@
...
-#endif
-
-stackSave = stackRestore = stackAlloc = function() {
-  abort('cannot use the stack before compiled code is ready to run, and has provided stack access');
-};
-
-function staticAlloc(size) {
-  abort('staticAlloc is no longer available at runtime; instead, perform static allocations at compile time (using makeStaticAlloc)');
-}
-#endif
```
Comment (kripken) [inline src/support.js:31]: these assertions are useful, why remove them?

File: tests/runner.py [hunk 2]
```diff
@@ -244,6 +244,7 @@ def create_test_file(name, contents, binary=False):
   'wasm3',
   'wasms',
   'wasmz',
+  'strict'
```
Comment (kripken) [inline tests/runner.py:247]: this will run it locally by default, I think? does everything pass now?

File: tests/test_core.py [hunk 3]
```diff
@@ -5406,6 +5406,7 @@ def test_fs_errorstack(self, js_engines=[NODE_JS]):
   @also_with_noderawfs
   def test_fs_llseek(self, js_engines=None):
     self.set_setting('FORCE_FILESYSTEM', 1)
+    self.emcc_args += ['-lnodefs.js']
```
Comment (kripken) [inline tests/test_core.py:5409]: should `@also_with_noderawfs` do this? that would fix other tests too maybe.

File: tests/runner.py (hunk 2 above)
Comment (sbc100) [reply tests/runner.py:247]: This change allows me to run `strict.foo` from the command line.

Without this the runner doesn't know about a suite called `strict`.

As far as I know this doesn't run strict tests unless a user asks to run it.  Is there some command line I don't know about that runs all test suites?

File: src/support.js (hunk 1 above)
Comment (sbc100) [reply src/support.js:31]: Reverted for now.

File: tests/test_core.py (hunk 3 above)
Comment (sbc100) [reply tests/test_core.py:5409]: reverted

File: tests/runner.py (hunk 2 above)
Comment (kripken) [reply tests/runner.py:247]: I was thinking about `./tests/runner.py *.test_loop`, but looks like that does even more, so I was wrong, sorry, this looks fine.

**Pull request merged**